backend/about.py

```diff
@@ -0,0 +1,92 @@
+# This file scrapes the official pyladies 
+# website 'https://pyladies.com/about/'
+# and assigns about info text to 
+# str type variable 'pyladies_about_scraped'
+# which can be further used in our website
+
+# in case the structure is of official pyladies
+# website is changed in future, link to the website
+# as str is assigned to 'pyladies_about_scraped' variable
+
+import os
+import requests
+from bs4 import BeautifulSoup   # for web scraping
+
+pyladies_url="https://pyladies.com/about/"
+
+paragraph1 = ""
+paragraph2 = ""
+paragraph3 = ""
+paragraph4 = ""
+
+def get_pyladies_about_scraping(url=pyladies_url):
+    """ scrapes official pyladies website;
+    returns pyladies_about_info str variable
+    with info about pyladies;
+    if the structure of the scraped website is changed,
+    assigns url address as str instead """
+
+    page = requests.get(pyladies_url)
+    soup = BeautifulSoup(page.content, 'html.parser')
+
+    # getting the desired text from scraped html page
+    div_class_page = soup.find("div", {"class": "page"})
+    pyladies_about_scraped = div_class_page.find("p")
+
+    if pyladies_about_scraped:
+        pyladies_about_scraped = pyladies_about_scraped.text.strip()
+    # in case the website's structure is changed
+    else:
+        pyladies_about_scraped= "Something went wrong :(. Please visit " + pyladies_url + " for the info from the international Pyladies website"
+
+    return pyladies_about_scraped
+
+
+dirname = os.path.dirname(__file__)
```

Comment:
this should not be needed, we should be able to know where we are and point to the correct directory where the file is.